custom-plugins/grafana-worldmap-panel/src/worldmap_ctrl.ts

```diff
@@ -299,6 +313,12 @@ export default class WorldmapCtrl extends MetricsPanelCtrl {
     }
   }
 
+  changeLinkedVariable() {
+    console.log("Updating linked variable...");
```

Comment:
should these be doing something?

Reply:
Actually, no. I thought this would be required to update the selected value, but it looks like Angular takes care of it using its `ng-model` attribute. Saving the change is still not working properly, so I've added a comment to this function explaining its existence -- it can probably be removed later, once we've got saving the change working.